### PR TITLE
feat(thumbnail): created a React version of the thumbnail component

### PR DESCRIPTION
**Pull request by GuillaumeNachury**
Created a React version of the thumbnail component

### CODE REVIEW

File: demo/react/layout/SubNav.tsx [hunk 1]
```diff
@@ -60,6 +60,7 @@ const NAV_ITEMS: string[] = [
     'Text field',
     'Toolbar',
     'Tooltip',
+    'Thumbnail',
```
Comment (mcastagnetti) [inline demo/react/layout/SubNav.tsx]: Alphabetic order ?

File: src/components/thumbnail/react/Thumbnail.tsx [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+ */
+const DEFAULT_PROPS: IDefaultPropsType = {
+    size: Sizes.m,
+    theme: Themes.light,
+    variant: Variants.squared,
+};
+/////////////////////////////
+
+/**
+ * [Enter the description of the component here].
```
Comment (mcastagnetti) [inline src/components/thumbnail/react/Thumbnail.tsx]: Little description to add here

File: demo/react/layout/SubNav.tsx (hunk 1 above)
Comment (GuillaumeNachury) [reply demo/react/layout/SubNav.tsx]: done.

File: src/components/thumbnail/react/Thumbnail.tsx (hunk 2 above)
Comment (GuillaumeNachury) [reply src/components/thumbnail/react/Thumbnail.tsx]: Good catch,
done.

**Comment by matmkian** [comment]
Test ok

**Pull request merged**